src/scss/main.scss

```diff
@@ -2,4 +2,6 @@
 @import "utils/variables";
 @import "utils/mixins";
 @import "utils/fonts";
-@import "utils/base.scss";
\ No newline at end of file
+@import "utils/base.scss";
```

Comment:
consider creating a folder for each component with index.js and {componentName}.scss

Reply:
done